src/pkg/packager/interactive.go

```diff
@@ -44,7 +45,25 @@ func (p *Packager) confirmAction(userMessage string, sbomViewFiles []string) (co
 
 	// Prompt the user for confirmation, on abort return false
 	if err := survey.AskOne(prompt, &confirm); err != nil {
-		return false
+		confirm = true
+	}
+
+	// On create in interactive mode, prompt for max package size
+	if userMessage == "Create" {
+		value, _ := p.promptVariable(types.ZarfPackageVariable{
+			Name:        "Maximum Package Size",
+			Description: "Specify a maximum file size for this package in Megabytes (MB). Above this size, the package will be split into multiple files. 0 will disable this feature.",
```

Comment:
Technically... This is Mebibytes (MiB) (1024*1024) not strictly Megabytes (MB) (1000*1000).  I know some of our personas might not know what that means, but we could hint at it by saying `Megabytes (MiB)` instead.

https://simple.wikipedia.org/wiki/Mebibyte
https://simple.wikipedia.org/wiki/Megabyte